src-ui/src/app/components/dashboard/widgets/saved-view-widget/saved-view-widget.component.html

```diff
@@ -13,7 +13,7 @@
     <tbody>
       <tr *ngFor="let doc of documents" routerLink="/documents/{{doc.id}}">
         <td>{{doc.created | customDate}}</td>
-        <td>{{doc.title | documentTitle}}<app-tag [tag]="t" *ngFor="let t of doc.tags$ | async" class="ms-1"></app-tag></td>
+        <td>{{doc.title | documentTitle}}<app-tag [tag]="t" *ngFor="let t of doc.tags$ | async" class="ms-1" (click)="clickTag(t); $event.stopPropagation();"></app-tag></td>
```

Comment:
if you put the router link on the `doc.title`, there is no need to call stopPropagation. It will reduce side-effects coming from event bubbling.

can you please change the router-link directive to `[routerLink]=['/','documents',doc.id]`? There is no need for interpolation here.

Reply:
Sure about `routerLink`, as for moving the link to doc.title, as is the entire `<tr>` row is a link to the document, which I think is preferable, as opposed to just the `<td>`

Reply:
okay, agree with that. Nevertheless, in the video, you clicked on the title 😄 .